practice-two/src/js/services/httpClient.service.js

```diff
@@ -0,0 +1,57 @@
+import axios from 'axios';
+
+export default class HttpService {
+  constructor() {
+    this.baseUrl = process.env.BASE_API_URL;
+    this.axiosClient = axios.create({
+      baseURL: this.baseUrl,
+      headers: {
+        'Content-Type': 'application/json'
+      },
+    });
+  }
+
+  async get(endpoint, params = {}) {
+    try {
+      const response = await this.axiosClient.get(`${endpoint}`, { params });
+
+      return response.data;
+    } catch (error) {
+      console.error(error.message);
+    }
+  }
+
+  async post(endpoint, product) {
```

Comment:
this should be `async post(endpoint, payload)`. HttpService is comon service to reuse for multiple cases. If you use it to post a student, so it's not product at all